eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -225,6 +222,101 @@ public function countImageReferences($uri)
         return (int) $statement->fetchColumn();
     }
 
+    /**
+     * Returns the number of recorded references outside of the given $path.
+     *
+     * @param string $uri File IO uri (not legacy)
+     *
+     * @return int
+     */
```

Comment:
CS: Please skip the doc, if the doc exists on parent (no inheritdoc needed also)